tests/unit/common/test_resources.py

```diff
@@ -0,0 +1,168 @@
+import pandas as pd
+
+from hdmf.common.resources import ExternalResources
+from hdmf.testing import TestCase, H5RoundTripMixin
+
+
+class TestExternalResources(H5RoundTripMixin, TestCase):
```

Comment:
These are all good tests for ExternalResources, but it would be ideal to have unit tests for the new Row, RowGetter, and Table classes as well.